scheduler/test/cook/test/rest/api.clj

```diff
@@ -434,6 +435,91 @@
               trimmed-body (select-keys body (keys successful-job))]
           (is (= (dissoc successful-job "uris") (dissoc trimmed-body "uris"))))))))
 
+(deftest disk-api
+  (setup)
+  (let [conn (restore-fresh-database! "datomic:mem://mesos-api-test")
+        _ (create-pool conn "test-pool")
+        job (fn [disk] (merge (basic-job) {"disk" disk}))
+        h (basic-handler conn)]
+    (testing "negative disk invalid"
+      (is (<= 400
+              (:status (h {:request-method :post
+                           :scheme :http
+                           :uri "/jobs"
+                           :headers {"Content-Type" "application/json"}
+                           :authorization/user "dgrnbrg"
+                           :body-params {"jobs" [(job {"size" -4})] "pool" "test-pool"}}))
+              499)))
```

Comment:
Instead of ranges, can we check for specific status codes on these?

Reply:
yeah, is there a reason why previous tests followed the convention with ranges?

Reply:
Hmm, not sure, but it seems better to me to be precise.

Reply:
updated